---
title: 'Nonpathogenic Bacterial Microbes Present in San Francisco Public Transportation – BART and Muni'
author: "Jacqueline Imperial"
date: "October 16, 2019"
output: word_document
csl: bioinformatics.csl
bibliography: references.bib
---

```{r setup, include = FALSE}
knitr::opts_chunk$set(echo = FALSE, message = FALSE, warning = FALSE)
```

# Introduction

Bacterial microbes present in urban locations have been a major public 
health issue especially during the peak of the flu season. Major concerns arise 
from the transfer of bacteria from person to person in public locations such as 
parks, schools, hospitals, and public transportation, all containing abundant 
bacteria [@reynolds2005occurrence]. These bacteria may pose a threat to the 
health of individuals due to a high probability of the bacteria being pathogenic [@onat2017relationship]. According to numerous studies, some pathogens found in public transit include the *Staphylococcus* genus[@stepanovic2008public], commonly found on human skin, and *S. aureus*, a pathogen responsible for giving rise to Methicillin-resistant *Staphylococcus aureus*, or MRSA [@otter2009bacterial]. MRSA can cause mild infections on the skin, but some exposure may be life-threatening [@carleton2004community]. Therefore, due to this public health concern, it is crucial to analyze the possibly dangerous strains of microbial communities present and growing in such locations.

The aim of my study is to shed light on the lack of cleanliness and 
maintenance done on public transportation in major cities such as San
Francisco’s Bay Area Rapid Transit (BART) and Municipal Railway (Muni).
San Francisco State University’s Biology Department has recently extracted
microbial communities in micro-locations of public transportation harboring 
abundant bacteria. These locations included hand-touch surfaces, such as 
handrails and doors, seats, and the floor. The findings of this particular study
found that the majority of bacteria was located on the floor [@yeh2011diversity]. Numerous other 
similar research has been done in in different cities such as London and Seoul 
in order to investigate the culturable airborne bacterial and fungal communities
in public transportation locations [@hwang2014comparison].  

Similar to the study previously mentioned, the purpose of my study was to 
examine and determine the types of bacterial communities harbored on the floor
of the BART and Muni. Particularly, my primary focal question for this study 
was “what kinds of bacteria contaminated these environments?”. Additionally, 
I wanted to know how drastically the diversity and abundance of the bacteria
differed between the two types of public transportation in the city area. 
Based on these two leading questions, I hypothesized that *Staphylococci* and 
other less harmful pathogens would be present in both locations. This 
hypothesis is derived from both the lack of cleanliness from my personal 
experiences using BART and Muni, and previous findings from similar studies. 
I also believe that the diversity will be larger in the Muni due to the lack
of maintenance and high volumes of passengers on a day-to-day basis. 

In order to test these hypotheses, I obtained 3 individual swab samples from 
each the Muni and BART floors. The 6 samples in total were diluted 1, 1:10, 
and 1:100 respectively and cultured on an agar dish. I extracted and amplified 
the DNA from the 6 most isolated colonies through PCR. Afterwards, I utilized 
gel electrophoresis in order to separate these DNA fragments according to the 
size and charge. Lastly, I used unidirectional Sanger sequencing, which allowed 
for the further analysis of the specific DNA composition. I compared the 
successful sequences with BLAST, which helped me determine what types of 
microbial strains were present within the colony samples. 

The primary findings of this study revealed that the Muni contained more 
colonies after culturing. These abundant colonies from the Muni were also 
more diverse in morphotypes compared to the cultured BART samples. According
to the molecular data, both BART 1 diluted 1:10 and 1:100 possessed
*Microbacterium* strains and BART 2 1:100 contained *Acinetobacter schindleri* 
strain. Muni 2 1:10 also contained a strain of *Acinetobacter* and Muni 3 1:100 
contained *Marinobacterium* [@takeuchi1998union].  


# Methods

## Sampling

Materials required for the extraction of samples are 12 sterile 1.5mL tubes and swabs, gloves, zip block bags, and sterile phosphate-buffered saline (PBS). I collected all the samples in one day. For the BART, I obtained the samples from the Richmond train, from three different carts. I extracted the Muni samples from three difference buses on the 5R route. I specifically obtained all samples from the entrances of all trains and buses, where foot-traffic is the greatest. I took two swab samples from each specific location. First, I dipped the sterile swab to moistened it into the sterile PBS before gathering the sample from the floor. I gently whipped the floor surface for 10 seconds with one end of the swab. I placed the swab into its designated labeled tube, and broke off the long end of the swab in order to fit it in the tube. Each BART and Muni samples were labeled clearly. 

## Culturing and DNA Extraction

24 hours after extraction,  I diluted and cultured the samples. Before plating, all 6 samples were diluted 1:1, 1:10, and 1:100. I created the dilutions with the samples and designated amounts of PBS. In order to culture all 18 samples and their dilutions, I pipetted 100μl of each sample onto a plate. I used 18 100mm TSA plates for plating samples from each of my 2 treatments (n=3 per group, 6 total). I spread the sample onto the surface with the use of rattler beads. All 18 samples were then incubated for approximately 4 days before the analysis of colony abundance and morphotypes. I then recorded my observations based on the culture abundance counts and number of distinguishable morphotypes. 
	
I extracted DNA from the cultures with the most isolated colonies. For this study, these 6 target colonies derived from B1 1:10, B1 1:100, B2 1:100, M2 1:10, M3 1:10, and M3 1:100. For DNA extraction, I followed the manufacturer’s protocol from the Sigma REDExtract-N-Amp kit [@weberextract]. Afterwards, I used Qubit to quantify the target colony DNA extractions [@haines2019detection]. To create the Qubit mix, I added 198μl of Qubit solution to each Qubit tube. I also pipetted 2μl of the DNA extraction, vortexed the mix for 5 seconds, and incubated the samples for approximately 5 minutes in the dark. The Qubit machine quantified each sample and I recorded the DNA concentrations (ng/μl).

## PCR and Gel Electrophoresis

During the following week, a PCR procedure was utilized in order to amplify the DNA extractions [@kuno1998universal]. Before beginning the protocol, I labeled a sterile tube for the master mix. I calculated the necessary volumes of reagents needed for the master mix using (n+1) + 10%, where n=7. The master mix contained; 77μl Amp, 6.16μl 27f primer, 6.16μl 1492r primer, 7.7μl BSA, and 49.28μl H_2 O. I aliquoted 19μl of the master mix into all sample tubes and negative control tube. I also pipetted 1μl of DNA extraction sample into its designated tube and 1μl H2O into to the negative control. The  automated PCR cycles ran overnight. The PCR ran at 95º C for 5 minutes in order to denature and separate the long DNA strands. The temperature was then lowered to 94º C for 30 seconds. Temperature was further lowered to 65º C for 30 seconds, which is critical for annealing and for the primers to sit on the DNA. The temperature was increased again to 72º C for 1 minute for extension and elongation by DNA polymerase. The temperature was raised to 95º C for 30 seconds, 55º C for 30 seconds, and 72º C for 1 minute for a total of 25 cycles. Afterwards, the temperature was held at 72º C for 10 minutes and held 4º C. The touchdown PCR contained 2% agarose gel, SYBR safe dye, and TAE buffer. After the PCR image was taken,  I recorded all observations based on the brightness or faintness of the band. Following the PCR, I ran the products on a gel electrophoresis gel at 140 volts for 30 minutes in order to separate the DNA fragments according to their size and charge [@aaij1972gel].  

## Sanger Sequencing

After checking for successful amplification from PCR utilizing gel electrophoresis, I cleaned the PCR products using Invitrogen Exo-SAP (Exonuclease – Shrimp Alkaline Phosphatase). They were then sent for unidirectional Sanger sequencing at MCLAB located in South San Francisco, CA [@blazej2006microfabricated]. The Sanger sequencing results were finalized and sent back the following day. 

### Sequence Analysis Methods

In order to clean and trim the sequences, I used Geneious software for data analysis. I first viewed all samples to determine which ones were successful and which can be cleaned and analyzed through BLAST. I then began the process of trimming the successful samples by taking out the beginning and ends of the sequence. I also cleaned the sequence data by using the IUPAC ambiguity codes to fixed those that contained mixed peaks. After cleaning and trimming the sequences, I used NCBI’s Nucleotide  BLAST to determine what strains the colony contained [@johnson2008ncbi]. All observations of the strains of each successful sample were recorded. 


# Results

After quantifying DNA PCR products with Qubit, DNA concentrations from BART significantly varied, ranging approximately 5.92-34.1 ng/μl. However, concentrations from Muni were consistent around 8.00-9.00 ng/μl (Table 1). According to the image of culture PCRs derived from the DNA extractions depicted Figure 1, the majority of the samples contain faint, but distinguishable, bands and ~300bp in reference to the ladder located in the far right lane (Figure 1).



| Sample I.D. | Sample Group      | Sample Dilution I.D. | DNA Concentration (ng/μl) |
|-------------|-------------------|----------------------|---------------------------|
| B1          | BART              | 10                   | 34.1                      |
| B1          | BART              | 100                  | 11.5                      |
| B2          | BART              | 100                  | 5.92                      |
| M2          | Municipal Railway | 10                   | 8.27                      |
| M3          | Municipal Railway | 10                   | 8.33                      |
| M3          | Municipal Railway | 100                  | 8.37                      |
**Table 1:** Table of culture Qubit DNA concentration results. BART DNA concentrations signficantly vary, however concentrations are sufficient for PCR.



![](images/Culture_PCR.jpg)
**Figure 1:** Image of culture PCRs from DNA extractions of most abundant and isolated colonies. The ladder indicates that the sequences are approximately 300bp.

```{r load-libraries}
library("ggplot2")
library("dplyr")
library("tidyr")
library("readr")
library("broom")
library("knitr")
```

Figure 2 shows a boxplot of the culture 10x dilution colony abundances in each treatments, BART and Muni. The statistic value is 3.5 and the Wilcox p-value is 0.82 for both treatments (Figure 2). Figure 3 also depicts a boxplot, but of the number of morphotypes in the 10x diluted colonies for both treatments. The statistic W value is 1.0 and the p-value is 0.16 (Figure 3).


```{r read-in-culture-data}
# read in the culture data from csv
culture_data <- read_csv("data/raw_data/raw_culture_count_data/BIOL422_culturing_data - Sheet1 (2).csv")
```

```{r filter-and-plot-abundances}
# filter out only my 100x dilutions and make a boxplot
culture_data %>%
  filter(student_name == "Jacqueline Imperial") %>% 
  filter(dilution_1_10_or_100 == "10") %>%
  ggplot(aes(x = sample_group,
             y = number_of_colonies)) +
  geom_boxplot() +
  ggtitle("Comparison of Colony Abundances Between BART and Municipal Railway",
                "10x Dilution") +
  xlab("Sample Group") +
  ylab("Number of Colonies")
```

**Figure 2:** Boxplot of colony abundances at different sites, 10x dilution. Despite a higher median number of colonies from the Municipal Railway samples, the mean values of colonies were not significantly different between the two sites (Wilcox p = 0.82)

```{r abundances-stats}
# run a statistical test to compare the two groups of abundances
culture_data %>%
  filter(student_name == "Jacqueline Imperial") %>% 
  filter(dilution_1_10_or_100 == "10") %>%
  wilcox.test(data = .,
              number_of_colonies ~ sample_group) %>%
  broom::glance(x = .) %>%
  knitr::kable()
```

```{r filter-and-plot-morphotypes}
# filter out only my 100x dilutions and make a boxplot
culture_data %>%
  filter(student_name == "Jacqueline Imperial") %>% 
  filter(dilution_1_10_or_100 == "10") %>%
  ggplot(aes(x = sample_group,
             y = as.numeric(number_of_morphotypes))) +
  geom_boxplot() +
  ggtitle("Comparison of Morphotypes Between BART and Municipal Railway",
                "10x Dilution") +
  xlab("Sample Group") +
  ylab("Number of Morphotypes")
```

**Figure 3:** Boxplot showing the number of morphotypes from the two different sites. There was a difference in the mean number of morphotypes, W = 1 and p = 0.16

```{r morphotypes-stats}
# run a statistical test to compare the two groups of abundances
culture_data %>%
  filter(student_name == "Jacqueline Imperial") %>% 
  filter(dilution_1_10_or_100 == "10") %>%
  wilcox.test(data = .,
              as.numeric(number_of_morphotypes) ~ sample_group) %>%
  broom::glance(x = .) %>%
  knitr::kable()
```


After Sanger sequencing, the samples’ sequences were cleaned and trimmed through Geneious. Table 2 reveals all samples’ loci are 16S. After analysis, B1 1:10, B1 1:100. B2 1:100, and M3 1:100 were determined usable and M2 1:10 and the negative control were not usable for the next step of BLAST analysis. Table 2 also indicated the lengths of the sequences after trimming and cleaning, and the manual corrections made (Table 2).


| Sequence Locus | Original Filename       | Filename after Correction                       | Usability | Length After Trimming | Number of Manual Corrections |
|----------------|-------------------------|-------------------------------------------------|-----------|-----------------------|------------------------------|
| 16S            | JI_B1_1_10_27f_A06.ab1  | JI_B1_1_10_27f_A06_JI_cleaned_mixed.ab1         | Usable    | 280                   | 1                            |
| 16S            | JI_B1_1_100_27f_B06.ab1 | JI_B1_1_100_27f_B06_JI_cleaned.ab1              | Usable    | 771                   | 1                            |
| 16S            | JI_B2_1_100_27f_C06.ab1 | JI_B2_1_100_27f_C06_JI_cleaned.ab1              | Usable    | 609                   | 4                            |
| 16S            | JI_M2_1_10_27f_D06.ab1  | JI_M2_1_10_27f_D06_JI_cleaned.ab1               | Usable    | 633                   | 1                            |
| 16S            | JI_M3_1_10_27f_E06.ab1  | JI_M3_1_10_27f_E06_JI_failed.ab1                | Unusable  | 5                     | 0                            |
| 16S            | JI_M3_1_100_27f_F06.ab1 | JI_M3_1_100_27f_F06_JI_cleaned_failed_blast.ab1 | Usable    | 824                   | 3                            |
| 16S            | JI_neg_27f_G06.ab1      | JI_neg_27f_G06_JI_failed.ab1                    | Unusable  | 5                     | 0                            |

**Table 2:** Sequence analysis after cleaning and trimming using Geneious. 


Figure 4 shows the Maximum Likelihood Phylogeny of the the successful samples. The bootstrap value is 100 and the posterior probability value is 0.06 (Figure 4). Figure 5 shows a Bayesian Phylogeny between these same samples. The bootstrap value is 1 and the Bayesian posterior probability is 0.07 (Figure 5).

![](output/phylogenies/sanger_sequences_phyml_phylogeny.png)
**Figure 4:** Maximum Likelihood Phylogeny of the three most successful samples with the root being *Thermus aquaticus*. M2 1:10 and B2 1:100 are more closely related than B1 1:100, with a bootstrap of 100. 

![](output/phylogenies/sanger_sequences_mrbayes.png)
**Figure 5:** A Bayesian Phylogeny with bootstrap 1 and Bayesian posterior probability of 0.07.

Lastly, the results of Nucleotide BLAST revealed which strains predominated within each sample colonies, indicated in Table 3. From this, I discovered that B1 1:10 contains *Macrobacterium phyllosphaerae* strain.  B1 1:100 contains *Microbacterium hydrocarbonoxydans* strain, B2 1:100 has *Acinetobacter schindleri* strain, M2 1:100 has *Acinetobacter sp.* strain, and M3 1:100 contains a *Marinobacterium sp* MB3 gene. M3 1:10 and the negative control were failed sequences (Table 3). 

| Sample I.D.      | Description                                                                             | Percent Identity | Query Cover | E-Value | Accession  |
|------------------|-----------------------------------------------------------------------------------------|------------------|-------------|---------|------------|
| B1 1:10          | Microbacterium phyllosphaerae strain Y137 16S ribosomal RNA gene, partial sequence      | 100%             | 100%        | 6e-143  | MN477985.1 |
| B1 1:100         | Microbacterium hydrocarbonoxydans strain T11.3 16S ribosomal RNA gene, partial sequence | 99.87%           | 99%         | 0.0     | MN198007.1 |
| B2 1:100         | Acinetobacter schindleri strain H3 chromosome, complete genome                          | 99.34%           | 100%        | 0.0     | CP030754.1 |
| M2 1:10          | Acinetobacter sp. strain YX3 16S ribosomal RNA gene, partial sequence                   | 99.84%           | 99%         | 0.0     | MK138620.1 |
| M3 1:10          | Sequence Failed                                                                         |                  |             |         |            |
| M3 1:100         | Marinobacterium sp. MB3 gene for 16S rRNA, partial sequence                             | 76.74%           | 36%         | 2e-38   | AB490785.1 |
| Negative Control | Sequence Failed                                                                         |                  |             |         |            |
**Table 3:** A table of BLAST results from Sanger Sequence samples. The table indicates which strains are present in the bacterial colonies. 

```{r your-code-here}
# Add code chunks as needed for your analyses
```

# Discussion

It’s evident that many forms of bacteria grow on the surface of the floor in public transportation locations, especially San Francisco’s BART and Muni. The main questions going into this study were to determine what kinds of bacterial strains contaminated the two environments and whether or not the diversity and abundance greatly differed between the two treatments. From these leading questions and in reference to previous similar studies, I hypothesized that *Staphylococci* and other less harmful pathogenic bacteria would be found in both BART and Muni. Furthermore, I hypothesized that the diversity would be significantly greater in the Muni samples compared to Bart samples due to the lack of maintenance and cleanliness of the city buses as well as the high volume of people going through that specific form of transportation.

According to my results, specifically the boxplot of the comparison between colony abundances between the two treatments in the 10x dilution in Figure 2, the abundances did not vary significantly. Despite the high median value of colony abundance in the Muni samples, the mean values are not significantly different. Also, because the p-value is 0.82, I fail to reject the null hypothesis which states that both treatment abundances are similar. Moreover, the boxplot depicted in Figure 3 shows a comparison between the morphotypes of the two treatments in the 10x dilution as well. Although both the mean and median vary greatly according to the boxplot, the p-value equals 0.16. Due to these results, I fail to reject the null hypothesis, which states that the morphotypes between BART and Muni are the same. Therefore, the results from Figure 2 and 3 reveal that the bacterial communities growing on the BART and Muni floor surfaces do not greatly differ in abundance and diversity.
	
Factors that may influence these findings may be that the same groups of people may be using both types of public transportation. Or the bacteria on the bottom of individuals’ shoes do not make a major difference in the bacteria found on the public transportation floor because much of the bacteria picked up from the ground within the Bay Area are transferred equally to both locations. Additionally, the maintenance of both the BART and Muni may be more similar that I thought before the study. Despite the apparent lack of cleanliness in the Muni, it is nearly equal in the BART as well. These factor may have caused the results found in the boxplots and may have led to the conclusion that the diversity and abundance of bacterial colonies in the BART and Muni do not differ greatly. These particular results align with much of the results found by other similar studies.
	
My second question for this study addressed the contents of the bacteria on the floor and what strains existed in these locations. After cleaning and trimming the Sanger Sequencing data and matching it with BLAST results, I discovered the taxa in such locations. According to Table 3 of my results, I found that the majority of taxa were *Microbacterium* and *Acinetobacter* strains. Both strains are nonpathogenic and mostly not harmful for human health. These taxa findings greatly differed from the findings of studies in London, where research has found *Staphylococcus aureus* on hand-touch surfaces in public transportation [@otter2009bacterial]. These findings also did not align with my initial hypothesis. I’m confident in the sample BLAST results that contained a percent identity of more than 90%. Therefore, I am confident in the BLAST results for B1 1:10, B1 1:100, B2 1:100, M2 1:10, and M3 1:10.

Factors that may influence that deviation from my taxa findings to previous studies such as the one done in London may be the difference in culture and overall location. London contains higher volumes of people of different cultures, both locals and tourists, compared to San Francisco, where most people opt to take a car to travel, rather than public transportation. Also the streets of London are less clean and maintained than the Bay Area. Lastly, the climate in the different places may also play a role in the growth of certain bacterial strains.

Additionally, the sequence for M3 1:10 was unusable for both phylogeny and BLAST. This may be due to a mistake in pipetting or extraction of colony DNA. It may also be because the colony was mixed with other neighboring colonies. Due to these possible factors, the sequence failed.

Overall, this study revealed that nonpathogenic bacterial communities grow on the floor public transportation in the San Francisco community. This indicates the improvement of maintenance within the city’s public locations. It also may reveal the future improvment of public health overall. 

# Sources Cited
